.gitignore

```diff
@@ -1,6 +1,35 @@
 # Created by https://www.toptal.com/developers/gitignore/api/visualstudio,reactnative,react,git,ruby,yarn,node,xcode,android,intellij,fastlane
```

Comment:
I expected this line to change @jai-charan

Reply:
like mentioning all sections that has been ignores?

Reply:
when you regenerate the page, please take the full block from the "starting" line to the "ending" line

Reply:
okay vijay